### ISSUE TITLE
something went wrong in documentation, on section: How to add a custom feature

### ISSUE THREAD

**Issue by tanlin2013** [blue-yonder/tsfresh#850]
**The problem**: In the section describing about how to add a custom feature, something was broken for combinatory features. It shows [(link to that page)](https://tsfresh.readthedocs.io/en/latest/text/how_to_add_custom_feature.html):

```
System Message: ERROR/3 (/home/docs/checkouts/readthedocs.org/user_builds/tsfresh/checkouts/latest/docs/text/how_to_add_custom_feature.rst, line 79)

Error in “code” directive: maximum 1 argument(s) allowed, 5 supplied.
```

**Anything else we need to know?**:

**Environment**:

- Python version:
- Operating System:
- tsfresh version:
- Install method (conda, pip, source):

**Comment by nils-braun** [comment]
Very nice catch!

**Comment by tanlin2013** [comment]
Actually, I just found 2 more with the keyword `system message`. You can also find them by searching this in the documentation. 😃

**Comment by nils-braun** [comment]
Something has clearly gone wrong here 😅
Thanks, would you like to do a PR or should I fix them?

**Comment by tanlin2013** [comment]
Sure, it's just a snap.
Implemented in PR [#853](https://github.com/blue-yonder/tsfresh/pull/853) 😃.
